packages/jest-snapshot/src/index.ts

```diff
@@ -26,6 +31,37 @@ type Context = MatcherState & {
   snapshotState: SnapshotState;
 };
 
+type MatchSnapshotConfig = {
+  context: Context;
+  expectedArgument: string;
+  inlineSnapshot?: string;
+  matcherName: string;
+  options: MatcherHintOptions;
+  propertyMatchers?: any;
+  received: any;
+  testName?: string;
+};
+
+const DID_NOT_THROW = 'Received function did not throw'; // same as toThrow
+const NOT_SNAPSHOT_MATCHERS = '.not cannot be used with snapshot matchers';
+
+// Display key in report when matcher fails same as in snapshot file,
+// but with optional name argument in green.
+const printKey = (path = '', name = '', count: number): string => {
+  const hasPath = path.length !== 0;
+  const hasName = name.length !== 0;
+
+  return (
+    '`' +
+    (hasPath ? utils.escapeBacktickString(path) : '') +
+    (hasPath && hasName ? ': ' : '') +
+    (hasName ? EXPECTED_COLOR(utils.escapeBacktickString(name)) : '') +
```

Comment:
I find coloring the name confusing. Since it doesn't matter in snapshot output, maybe we should use a different color, like yellow or cyan? This way it would be more obvious that the "name" in snapshot hint refers just to the snapshot key, not the error message

Reply:
Yes, after more discussion, we decided to use bold font weight.